src/OpenTelemetry.Instrumentation.AWSLambda/Implementation/AWSLambdaUtils.cs

```diff
@@ -88,17 +88,17 @@ internal static string GetCloudProvider()
 
     internal static string GetAWSRegion()
     {
-        return Environment.GetEnvironmentVariable(AWSRegion);
+        return Environment.GetEnvironmentVariable(AWSRegion) ?? string.Empty;
```

Comment:
Are you sure this is safe? Wouldn't it be safer to use `string?` as return value? Or did you check all callers?

Reply:
In Lambda the AWS_REGION environment variable will never not be set but in a test environment it could not be set. I'm with @Oberon00 that changing the return type to `string?` is the safer choice and more in the spirit of nullability.